### PR TITLE
Add escaped object to simple primitives.

### PR DESCRIPTION
**Pull request by Yeti-or**

### CODE REVIEW

**Comment by miripiruni** [comment]
@Yeti-or can you provide background info, please?

File: lib/bemxjst/utils.js [hunk 1]
```diff
@@ -51,6 +51,10 @@ exports.isShortTag = function isShortTag(t) {
 
 exports.isSimple = function isSimple(obj) {
   if (!obj || obj === true) return true;
+  if (!obj.block &&
```
Comment (miripiruni) [inline lib/bemxjst/utils.js]: Add `!obj.elem` please

**Comment by Yeti-or** [comment]
Idea of this pr:
we have several *simple* objects, including `String`
but when we want to use `EsapedString` it become *complex* object.
Case for this idea:
We want to check for *simple* values in templates. But we need to keep in mind that escaped strings are not simple.
And when we operate this values in templates (for example strings and escaped strings) in most cases we use them similarly.
I think that it would be easier in templates to use just `isSimple` helper, or to create another helper `isSimpleValue` for example

**Comment by Yeti-or** [comment]
@zxqfox what are you thinking about that?

**Comment by miripiruni** [comment]
@Yeti-or OK. Good.

But Travis failed. And see https://github.com/bem/bem-xjst/pull/290#discussion_r67190873

**Comment by miripiruni** [comment]
LGTM

File: lib/bemxjst/utils.js [hunk 2]
```diff
@@ -52,6 +52,10 @@ exports.isShortTag = function isShortTag(t) {
 exports.isSimple = function isSimple(obj) {
   if (!obj || obj === true) return true;
   if (!obj.block &&
+      !obj.elem &&
+      !obj.tag &&
+      !obj.cls &&
+      !obj.attrs &&
       obj.hasOwnProperty('html') &&
       isSimple(obj.html))
```
Comment (zxqfox) [inline lib/bemxjst/utils.js]: Я бы порядок сменил, потому что `obj.html` намного чаще будет возвращать `false`.

**Pull request merged**